tox.ini

```diff
@@ -14,7 +14,9 @@ deps =
   pycmd
   # to avoid .eggs
   setuptools_scm
-  pytest30: pytest~=3.0.5
+  pytest30: pytest~=3.0.0
+  pytest31: pytest~=3.1.0
+  pytestmaster: git+https://github.com/pytest-dev/pytest@master
```

Comment:
Not sure if this is a good idea or not, thought I would open the PR directly instead of raising an issue;